requirements.txt

```diff
@@ -17,45 +19,3 @@ awscli-cwlogs>=1.4,<1.5
 
 git+https://github.com/cds-snc/notifier-utils.git@34.0.11#egg=notifications-utils==34.0.11
 
-## The following requirements were added by pip freeze:
```

Comment:
Looks like you need to re-run the freeze requirements again.

Might be the sh / bash thing.